RealmModelGenerator/JavaContentGenerator.swift

```diff
@@ -0,0 +1,208 @@
+        }
+        
+        content += " *\tModel file Generated using Realm Model Genrator.\n"
+        content += " */\n\n"
+    }
+    
+    //MARK: Append RealmKeys
+    func appendRealmKeys() {
+        content += "    public static final class RealmKeys {"
+        
+        for (_, (_, attr)) in entity.attributesByName.enumerate() {
+            var realmKey = "\n"
+            realmKey += "        public static final String " + attr.name.uppercaseString + "_KEY = \"" + attr.name.lowercaseString + "\";"
+            content += realmKey
+        }
+        
+        content += "\n    }\n\n"
+    }
+    
+    //MARK: Append attributes and relicationships
+    func appendAttributes()
+    {
+        let primarykey = entity.primaryKey
+        
+        // Append attributes
+        for (_, (_, attr)) in entity.attributesByName.enumerate() {
+            var attrAnnotions = ""
+            var attrDefination = ""
+            
+            //Get attribute annotions
+            if primarykey?.name == attr.name {
+                attrAnnotions += "@PrimaryKey\n"
+            }
+            
+            if attr.isIndexd && primarykey?.name != attr.name {
+                attrAnnotions += "@Index\n"
+            }
+            
+            if attr.isIgnored {
+                attrAnnotions += "@Ignore\n"
+            }
+            
+            if attr.isRequired {
+                attrAnnotions += "@Require\n"
+            }
+            
+            //Get attribute defination
+            if attr.hasDefault {
+                attrDefination += "private " + attr.type.rawValue + " " + attr.name + " = "
+                    + attr.defaultValue + ";"
+            } else {
+                attrDefination += "private " + attr.type.rawValue + " " + attr.name + ";"
+            }
+            
+            if !attrAnnotions.isEmpty {
+                content += "    " + attrAnnotions
+            }
+            content += "    " + attrDefination
+            content += "\n"
+        }
+        
+        // Append relationship
+        for (_, relationship) in entity.relationshipsByName {
+            var relationshipDefination = ""
+            if relationship.isMany {
+                relationshipDefination += "private RealmList<" + relationship.destination!.name + "> " + relationship.name
+            } else {
+                relationshipDefination += "private " + relationship.destination!.name + " " + relationship.name
+            }
+            content += "    " + relationshipDefination + "\n";
+        }
+        
+        content += "\n"
+    }
+    
+    //MARK: Append Getters and Setters
+    func appendGettersAndSetters()
+    {
+        // Append attribute Getters and Setters
+        for (_, (_, attr)) in entity.attributesByName.enumerate() {
+            var getter = "    "
+            var setter = "    "
+            
+            getter += "public " + attr.type.rawValue + " get" + attr.name.uppercaseFirst + "(){\n"
```

Comment:
should probably have a function that converts AttributeType to actually java/objc/swift types than relying on rawValue